### PR TITLE
Fix PR2 calibration by using hacked version of urdf_parser_py by overwriting PYTHONPATH in the shell scripts.

### PR DESCRIPTION
**Pull request by garaemon**
- add pr2_urdf_parser_py directory by copying from https://github.com/ros/urdfdom/pull/55
- overwrite PYTHONPATH when running multi_step_cov_estimator.py to use urdf_parser_py under pr2_urdf_parser_py directory instead of official urdf_parser_py
- You can test calibration by https://drive.google.com/file/d/0B4bnoIcbRJVuVWVqT1ExR0hvcEE/view?usp=sharing

### CODE REVIEW

File: pr2_calibration_launch/capture_data/capture_exec_pr2.launch [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<launch>
+<<launch>
```
Comment (mikeferguson) [inline pr2_calibration_launch/capture_data/capture_exec_pr2.launch]: this is probably wrong

Comment (garaemon) [inline pr2_calibration_launch/capture_data/capture_exec_pr2.launch]: fixed

**Comment by UltronDestroyer** [comment]
I'll give this a test on our PR2 and get back to this thread in 5 minutes.

**Comment by UltronDestroyer** [comment]
This code is copied from the old urdfdom, right?

Edit: nevermind, I should read more before replying

**Comment by garaemon** [comment]
nop, copied from the pull request nit merged into urdfdom by @k-okada

2014年12月2日火曜日、Devon Ashnotifications@github.comさんは書きました:

> This code is copied from the old urdfdom, right?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/PR2/pr2_calibration/pull/6#issuecomment-65234698.

## 

from iPhone

**Comment by UltronDestroyer** [comment]
Just to confirm to those who are interested: the pr2_recalibrate.launch process is not the calibration process that is under issue.

I've tested this on our robot and it works. Have you tried this on your robot @garaemon ?

**Comment by garaemon** [comment]
> Just to confirm to those who are interested: the pr2_recalibrate.launch process is not the calibration process that is under issue.

I don't know pr2_recalibrate.launch launch file. where can I find it?

**Comment by garaemon** [comment]
about estimate_pr2_beta_urdf.sh, it works on my environment.

**Comment by UltronDestroyer** [comment]
pr2_recalibrate.launch exists inside pr2_bringup

/opt/ros/hydro/share/pr2_bringup/

I'm not sure how it's different from pr2_calibration_launch.. The pr2_calibration_launch seems to be what is used to generate the initial URDF and the pr2_recalibrate is to.. fine tune later?

**Comment by garaemon** [comment]
I didn't know that.

> The pr2_calibration_launch seems to be what is used to generate the initial URDF and the pr2_recalibrate is to.. fine tune later?

I think we need to run pr2_recablirate before capturing data only if encode has large error.
But I think the encoders has good precision according to the calibration when starting up pr2.

**Comment by UltronDestroyer** [comment]
This is fixed now yeah?

**Comment by garaemon** [comment]
Yes, fixed

**Comment by UltronDestroyer** [comment]
Thanks  @garaemon

**Pull request merged**